template/package.json

```diff
@@ -16,7 +16,9 @@
     "sass": "^1.20.1",
     "tram-blueprints": "^1.0.0",
     "tram-one": "^8.1.1",
-    "xo": "^0.23.0"
+    "xo": "^0.23.0",
+    "eslint-config-tram-one": "^3.0.0",
+    "parcel-plugin-nuke-dist": "^1.0.0"
```

Comment:
this looks like it's not ordered correctly... it should be alphabetical (running `npm install` will actually put them in the order they should be as a reference).

Reply:
check again @JRJurman . i add `package-lock.json` to `.gitignore` also